### PR TITLE
Clarify XPath usage in DOMNode::C14N(File)

### PR DESCRIPTION
**Pull request by nielsdos**

### CODE REVIEW

File: language-snippets.ent [hunk 1]
```diff
@@ -1662,6 +1662,25 @@ it is inserted with (e.g.) <function xmlns="http://docbook.org/ns/docbook">DOMNo
  <classname xmlns="http://docbook.org/ns/docbook">DOMException</classname> with code
  <constant xmlns="http://docbook.org/ns/docbook">DOM_HIERARCHY_REQUEST_ERR</constant>.'>
 
+<!ENTITY dom.c14n.xpath_array '<listitem xmlns="http://docbook.org/ns/docbook">
+  <para>
+   An array of XPaths to filter the nodes by.
+   Each entry in this array is an associative array with:
+   <itemizedlist>
+    <listitem>
+     <para>
```
Comment (Girgias) [inline language-snippets.ent:1671]: Could you use `<simpara>` tags instead of the `<para>` ones within the listitem?

**Review by Girgias — APPROVED** [review]
LGTM, only one suggestion.

File: language-snippets.ent (hunk 1 above)
Comment (nielsdos) [reply language-snippets.ent]: Done. What's the actual difference in practice? I googled it and it seems like para but with more restrictions.

Comment (Girgias) [reply language-snippets.ent]: Basically, but also PhD will always use an HTML `<p>` tag rather than using a `<div>` styled as a paragraph

**Pull request merged**